### DISCUSSION TITLE
[@types/bootstrap]: popperConfig typings are wrong

### DISCUSSION THREAD

**Discussion by cdauth** [DefinitelyTyped/DefinitelyTyped#67333] [Issues with a @types package] [answered]
I've noticed that the typing for the `popperConfig` attribute is wrong in at least two places:

`Dropdown` has the following typings:
```typescript
popperConfig: Partial<Popper.Options> | PopperConfigFunction | null;
type PopperConfigFunction = () => Partial<Popper.Options>;
```

The `PopperConfigFunction` type is missing the `defaultBsPopperConfig: Popper.Options` parameter.

`Popover` has the following typings:
```typescript
popperConfig: Partial<Options> | PopperConfigFunction | null;
type PopperConfigFunction = (defaultBsPopperConfig: Options) => Partial<Options>;
```

Both types use `Options` (which refers to the popover options) instead of `Popper.Options`.

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
I assume it was corrected in #67334 (though I added a comment about the change internals)

**Reply by cdauth** [reply]
Brilliant, thanks for the quick fix!
